### PR TITLE
Add a flexbox demo

### PR DESCRIPTION
**Pull request by blink1073**
flexboxx-demo

### CODE REVIEW

File: examples/flexbox/LICENSE [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2014-2015, S. Chris Colbert
```
Comment (sccolbert) [inline examples/flexbox/LICENSE]: Wrong copyright header

File: examples/flexbox/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+flexbox-interrop
```
Comment (sccolbert) [inline examples/flexbox/README.md]: is this README needed? (i don't think it is)

File: examples/flexbox/LICENSE (hunk 1 above)
Comment (blink1073) [inline examples/flexbox/LICENSE]: Who is that damn Colbert guy clogging up our copyrights...

Comment (sccolbert) [inline examples/flexbox/LICENSE]: Actually, the license should be moved to the root directory to cover everything.

**Pull request merged**